### PR TITLE
Bump Flick to v1.7.0

### PR DESCRIPTION
**Pull request by saket**
Trying out a simpler way to setup Flick. Seems to be working fine with CatchUp.

### CODE REVIEW

File: app/src/main/kotlin/io/sweers/catchup/ui/activity/ImageViewerActivity.kt [hunk 1]
```diff
@@ -268,6 +260,14 @@ class ImageViewerActivity : AppCompatActivity() {
   }
 }
 
+private fun Context.dip(units: Int): Int {
+  return TypedValue.applyDimension(
+      COMPLEX_UNIT_DIP,
+      units.toFloat(),
+      resources.displayMetrics
+  ).toInt()
+}
```
Comment (saket) [inline app/src/main/kotlin/io/sweers/catchup/ui/activity/ImageViewerActivity.kt]: Do you prefer keeping the dimension in Xml?

File: app/src/main/kotlin/io/sweers/catchup/ui/activity/ImageViewerActivity.kt [hunk 2]
```diff
@@ -173,20 +175,10 @@ class ImageViewerActivity : AppCompatActivity() {
...
-        return when {
-          imageView.drawable == null -> resources.getDimensionPixelSize(
-              R.dimen.media_image_height_when_empty)
-          else -> imageView.visibleZoomedImageHeight.toInt()
-        }
-      }
+    val contentSizeProvider = ContentSizeProvider2 {
+      // A non-zero height is important so that the user can dismiss even
+      // the image is unavailable and the progress indicator is visible.
+      maxOf(dip(240), imageView.zoomedImageHeight.toInt())
```
Comment (saket) [inline app/src/main/kotlin/io/sweers/catchup/ui/activity/ImageViewerActivity.kt]: On a second thought, `imageView.zoomedImageHeight() ?: dip(240)` might be easier to understand. Can change if you want.

File: app/src/main/kotlin/io/sweers/catchup/ui/activity/ImageViewerActivity.kt (hunk 1 above)
Comment (ZacSweers) [reply app/src/main/kotlin/io/sweers/catchup/ui/activity/ImageViewerActivity.kt]: Nope I'm fine with this!

File: app/src/main/kotlin/io/sweers/catchup/ui/activity/ImageViewerActivity.kt (hunk 2 above)
Comment (ZacSweers) [reply app/src/main/kotlin/io/sweers/catchup/ui/activity/ImageViewerActivity.kt]: Does zoomedImageHeight() return a nullable?

Comment (saket) [reply app/src/main/kotlin/io/sweers/catchup/ui/activity/ImageViewerActivity.kt]: No, not yet

Comment (ZacSweers) [reply app/src/main/kotlin/io/sweers/catchup/ui/activity/ImageViewerActivity.kt]: Not yet? o_O

Comment (saket) [reply app/src/main/kotlin/io/sweers/catchup/ui/activity/ImageViewerActivity.kt]: Heh I meant it could be changed to do that

Comment (ZacSweers) [reply app/src/main/kotlin/io/sweers/catchup/ui/activity/ImageViewerActivity.kt]: Is that something we want? I'm mostly confused on why the comment is here if it's not something currently happening

Comment (saket) [reply app/src/main/kotlin/io/sweers/catchup/ui/activity/ImageViewerActivity.kt]: Na, let's ignore this comment. :)

**Comment by ZacSweers** [comment]
:+1:

**Pull request merged**